src/utilities/getTranslation.ts

```diff
@@ -0,0 +1,19 @@
+import type { i18n as Ii18n } from 'i18next';
+
+export const getTranslation = (label: Record<string, string> | string | JSX.Element, i18n: Ii18n): string => {
+  if (typeof label === 'object') {
+    if (label[i18n.language]) {
+      return label[i18n.language];
+    }
+    let fallbacks = [];
+    if (typeof i18n.options.fallbackLng === 'object') {
+      fallbacks = Object.keys(i18n.options.fallbackLng);
+    } else if (Array.isArray(i18n.options.fallbackLng)) {
+      fallbacks = typeof i18n.options.fallbackLng === 'string' ? [i18n.options.fallbackLng] : i18n.options.fallbackLng;
```

Comment:
An array is also typeof `object`, so this second case is unreachable, consider switching them:

```suggestion
    if (Array.isArray(i18n.options.fallbackLng)) {
      fallbacks = typeof i18n.options.fallbackLng === 'string' ? [i18n.options.fallbackLng] : i18n.options.fallbackLng;
    } else if (typeof i18n.options.fallbackLng === 'object') {
      fallbacks = Object.keys(i18n.options.fallbackLng);
```

Can `fallbackLng` be null? That would throw here

Reply:
Good catch!

Reply:
There is a default value for fallbackLng as 'en' from Payload here: src/translations/defaultOptions.ts 
For this to be null somebody would have to intentionally have set the `fallbackLng: null` in their own config i18n. I can't think of a reason why anyone would want to do that.